doc/rados/operations/erasure-code-clay.rst

```diff
@@ -181,7 +181,7 @@ During repair of an OSD, the helper information requested
 from an available OSD is only a fraction of a chunk. In fact, the number
 of sub-chunks within a chunk that are accessed during repair is given by:
 
-	repair sub-chunk count = sub-chunk count / q
+	repair sub-chunk count =  sub-chunk count / q
```

Comment:
@mynaramana why do you consider there is a typo in existing doc? and why shall we fix it?